src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,52 @@
 package mate.jdbc;
 
+import java.sql.SQLException;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+import mate.jdbc.util.ConnectionUtil;
+
 public class Main {
-    public static void main(String[] args) {
-        // test your code here
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
+    public static void main(String[] args) throws SQLException {
```

Comment:
you need catch these exceptions 
```suggestion
    public static void main(String[] args) {
```